doc/nrf/gs_programming.rst

```diff
@@ -91,7 +91,7 @@ You can download it from the following links:
       a. Select your project in the Project Explorer.
```

Comment:
@tejlmand  I know that this section probably needs some more changes but I wasn't sure which steps exactly are affected. Can you help me by leaving comments?